### PR TITLE
Add very simple support for fish and dolphin entities

### PR DESCRIPTION
**Pull request by kdurrance**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

Very simple base support for fish and dolphin entities, which will allow egg spawning underwater for instance. Does not support Drops yet.
...

Master
...

## Optional information
...

### CODE REVIEW

File: src/revivalpmmp/pureentities/data/MobTypeMaps.php [hunk 1]
```diff
@@ -39,7 +39,12 @@ class MobTypeMaps{
 	);
 
 	const PASSIVE_WET_MOBS = array(
-		"squid"
+		"squid",
+		"salmon",                                                                                                                   
+                "pufferfish",
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/data/MobTypeMaps.php]: formatting

File: src/revivalpmmp/pureentities/data/MobTypeMaps.php [hunk 2]
```diff
@@ -67,4 +72,4 @@ class MobTypeMaps{
 		"wither_skeleton",
 		"zombie_pigman"
 	);
-}
\ No newline at end of file
+}
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/data/MobTypeMaps.php]: no ending lines

File: src/revivalpmmp/pureentities/data/BiomeInfo.php [hunk 3]
```diff
@@ -73,4 +73,4 @@ class BiomeInfo{
 		Data::NETWORK_IDS["spider"],
 		Data::NETWORK_IDS["zombie"]
 	];
-}
\ No newline at end of file
+}
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/data/BiomeInfo.php]: no ending lines

File: src/revivalpmmp/pureentities/PureEntities.php [hunk 4]
```diff
@@ -486,4 +496,4 @@ public static function getPositionNearPlayer(Player $player, int $minimumDistanc
 
 		return new Position($x, $player->y, $z, $player->getLevel());
 	}
-}
\ No newline at end of file
+}
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/PureEntities.php]: no ending lines

File: src/revivalpmmp/pureentities/data/MobTypeMaps.php (hunk 2 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/data/MobTypeMaps.php]: I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

File: src/revivalpmmp/pureentities/data/BiomeInfo.php (hunk 3 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/data/BiomeInfo.php]: I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

File: src/revivalpmmp/pureentities/PureEntities.php (hunk 4 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/PureEntities.php]: I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

File: src/revivalpmmp/pureentities/data/MobTypeMaps.php (hunk 1 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/data/MobTypeMaps.php]: Fixed, should be good now

File: src/revivalpmmp/pureentities/data/MobTypeMaps.php (hunk 2 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/data/MobTypeMaps.php]: Confirmed, no empty new line at end of file?

File: src/revivalpmmp/pureentities/PureEntities.php (hunk 4 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/PureEntities.php]: Confirmed, no empty new line at end of file?

File: src/revivalpmmp/pureentities/data/BiomeInfo.php (hunk 3 above)
Comment (kdurrance) [reply src/revivalpmmp/pureentities/data/BiomeInfo.php]: Confirmed, no empty new line at end of file?

**Comment by jasonwynn10** [comment]
please rebase onto master, then we can merge

**Comment by kdurrance** [comment]
@jasonwynn10 just rebased, let me know if ok.

File: src/revivalpmmp/pureentities/data/BiomeInfo.php [hunk 5]
```diff
@@ -49,7 +49,7 @@ class BiomeInfo{
 		Biome::MOUNTAINS => [Data::NETWORK_IDS["llama"]],            // Extreme Hills
 		14 => [Data::NETWORK_IDS["mooshroom"]],                            // Mushroom Island
 		15 => [Data::NETWORK_IDS["mooshroom"]],                            // Mushroom Island Shore
-		Biome::OCEAN => [Data::NETWORK_IDS["squid"]],
+		Biome::OCEAN => [Data::NETWORK_IDS["squid"]], [Data::NETWORK_IDS["salmon"]], [Data::NETWORK_IDS["pufferfish"]], [Data::NETWORK_IDS["tropicalfish"]], [Data::NETWORK_IDS["cod"]], [Data::NETWORK_IDS["dolphin"]],
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/data/BiomeInfo.php]: please fix this array

Comment (jasonwynn10) [reply src/revivalpmmp/pureentities/data/BiomeInfo.php]: they should all be in one array like the line below it rather than multiple.

**Pull request merged**